toolkits/global/packages/global-article/__tests__/authors.js

```diff
@@ -65,26 +63,26 @@ describe('Authors', () => {
 		test('Should be able to create a new author list component', () => {
 			expect(authorList.create).toBeDefined();
 		});
-	
+
 		test('Should avoid jumping to the selected anchored section when js enabled', () => {
-			authorList.create($(selector));
-			expect(selector.querySelector('a').classList.contains('js-no-scroll')).toBeTruthy();
-		});	
-	
+			authorList.create(selector);
+			expect(selector.classList.contains('js-no-scroll')).toBeTruthy();
+		});
+
 		test('Should avoid the hidden affiliation links get keyboard focus', () => {
 			addSupElement();
-	
-			authorList.create($(selector));
-	
+
+			authorList.create(selector);
+
 			expect(selector.querySelector('sup a').getAttribute('tabIndex')).toBe('-1');
 		});
-	
+
 		test('Should register click events for each author name', () => {
-			jest.spyOn($.fn, 'delegate');
-	
-			authorList.create($(selector));
-	
-			expect($('.c-author-list').delegate).toHaveBeenCalled();
+			selector.addEventListener = jest.fn();
```

Comment:
This doesn't really check that it registers click events, just that the eventListener method was called